tests/assets/ansible/roles/caasp/tasks/Suse.yaml

```diff
@@ -35,7 +41,7 @@
   zypper:
       name:
         - skuba
```

Comment:
a better way would be to remove these lines and install

patterns-caasp-Management

which automatically pulls in the right versions of everything (no matter which caasp version)

Reply:
Interesting I was not aware of that package.

Is it not to heavy though ? I mean we do not want much more than kubectl and skuba in the end.

If not I'll update the ansible code.

Reply:
well, it does pull in a few terraform things, but I think that shouldn't matter?

Reply:
@brunoleon are you going to address the comment? Or keep it as is?

Reply:
I had to add public_cloud module because terraform provider lives there.
Should be good to merge.